packages/open-api-gateway/src/construct/utils/api-gateway-integrations-types.ts

```diff
@@ -57,7 +70,6 @@ export type OperationLookup = {
 
 /**
  * Auth types supported by the construct
- * TODO: Support cognito auth!
```

Comment:
fyi: You can move this as an inline `//` if you wish to keep it.

Reply:
Cool I'll just keep in my head for now - it's what I plan on working on once this is merged :)